src/wp-includes/formatting.php

```diff
@@ -2758,7 +2758,7 @@ function trailingslashit( $string ) {
  * @return string String without the trailing slashes.
  */
 function untrailingslashit( $string ) {
-	return rtrim( $string, '/\\' );
+	return is_string( $string ) ? rtrim( $string, '/\\' ) : '';
```

Comment:
Okay, so we have an issue when integers are passed. What do you think about something like this instead?
```suggestion
	if ( ! is_string( $string ) && ! is_numeric( $string ) ) {
		return '';
	}
	return rtrim( $string, '/\\' );
```

Reply:
Actually @DevHaoZi, now that I think about it, I think the original `(string)` cast makes more sense here.

If an `array` is passed, we can just let PHP's native error handling continue to do its job.

Implementing my previous suggestion(s) would change the behaviour of `untrailingslashit()` by not erroring out when it should.